packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -192,27 +209,28 @@ export function DifferentialAbundanceConfiguration(
   const collectionVarItems = useMemo(() => {
     return collections
       .filter((collectionVar) => {
-        return collectionVar.normalizationMethod
-          ? !collectionVar.isProportion &&
-              collectionVar.normalizationMethod === 'NULL' &&
-              !collectionVar.displayName?.includes('pathway')
+        return collectionVar.normalizationMethod // i guess diy stuff doesnt have this prop?
```

Comment:
It looks like the newly installed ones do, but just in case someone tries from an old analysis or something